stanford_news.install

```diff
@@ -0,0 +1,19 @@
+<?php
+
+/**
+ * @file
+ * Install Tasks.
+ */
+
+/**
+ * implments hook_install().
+ */
+function stanford_news_install() {
+  // Disable the core taxonomy_term view if it is enabled.
+  $core_taxonomy_view = \Drupal::configFactory()->get('views.view.taxonomy_term');
+  if ($core_taxonomy_view->get('status') == TRUE) {
+    $editable = \Drupal::configFactory()->getEditable('views.view.taxonomy_term');
```

Comment:
you could use `getEditable` in the first call, then you don't have to fetch it again. or better yet you could load the view entity instead.